frontend/demo/component/card/react/card-header-suffix.tsx

```diff
@@ -0,0 +1,20 @@
+import { reactExample } from 'Frontend/demo/react-example'; // hidden-source-line
+import React from 'react'; // hidden-source-line
+import { Card } from '@vaadin/react-components/Card.js';
+
+function Example() {
+  return (
+    // tag::snippet[]
+    <Card>
+      <div slot="title">Lapland</div>
+      <div slot="subtitle">The Exotic North</div>
+      <span slot="header-suffix" theme="badge success">
```

Comment:
Should use the following syntax to fix React example compilation:

```suggestion
      <span slot="header-suffix" {...{ theme: 'badge success' }} >
```

Reply:
Fixed.